Fix ci error

### Self Checks

- [x] I have read the [Contributing Guide](https://github.com/langgenius/dify/blob/main/CONTRIBUTING.md) and [Language Policy](https://github.com/langgenius/dify/issues/1542).
- [x] This is only for refactoring, if you would like to ask a question, please head to [Discussions](https://github.com/langgenius/dify/discussions/categories/general).
- [x] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [x] I confirm that I am using English to submit this report, otherwise it will be closed.
- [x] 【中文用户 & Non English User】请使用英语提交，否则会被关闭 ：）
- [x] Please do not modify this template :) and fill in all the required fields.

### Description

=================================== FAILURES ===================================
_________ test_apply_display_status_filter_applies_when_status_present _________

    def test_apply_display_status_filter_applies_when_status_present():
        query = sa.select(Document)
        filtered = DocumentService.apply_display_status_filter(query, "queuing")
        compiled = str(filtered.compile(compile_kwargs={"literal_binds": True}))
        assert "WHERE" in compiled
>       assert "document.indexing_status = 'waiting'" in compiled
E       assert "document.indexing_status = 'waiting'" in "SELECT documents.id, documents.tenant_id, documents.dataset_id, documents.position, documents.data_source_type, docum...oc_metadata, documents.doc_form, documents.doc_language \nFROM documents \nWHERE documents.indexing_status = 'waiting'"

api/tests/unit_tests/services/test_document_service_display_status.py:26: AssertionError

### Motivation

_No response_

### Additional Context

_No response_

--- a/api/tests/unit_tests/services/test_document_service_display_status.py
+++ b/api/tests/unit_tests/services/test_document_service_display_status.py
@@ -23,7 +23,7 @@ def test_apply_display_status_filter_applies_when_status_present():
     filtered = DocumentService.apply_display_status_filter(query, "queuing")
     compiled = str(filtered.compile(compile_kwargs={"literal_binds": True}))
     assert "WHERE" in compiled
-    assert "document.indexing_status = 'waiting'" in compiled
+    assert "documents.indexing_status = 'waiting'" in compiled
 
 
 def test_apply_display_status_filter_returns_same_when_invalid():

